commitlint.config.js

```diff
@@ -1 +1,131 @@
-module.exports = { extends: ["@commitlint/config-conventional"] };
+module.exports = {
+  rules: {
+    "body-leading-blank": [1, "always"],
+    "body-max-line-length": [2, "always", 100],
+    "footer-leading-blank": [1, "always"],
+    "footer-max-line-length": [2, "always", 100],
+    "header-max-length": [2, "always", 100],
+    "subject-case": [2, "never", ["pascal-case", "upper-case"]],
+    "subject-empty": [2, "never"],
+    "subject-full-stop": [2, "never", "."],
+    "type-case": [2, "always", "lower-case"],
+    "type-empty": [2, "never"],
+    "type-enum": [
+      2,
+      "always",
+      [
+        "build",
+        "chore",
+        "ci",
+        "docs",
+        "feat",
+        "fix",
+        "perf",
+        "refactor",
+        "revert",
+        "style",
+        "test",
+        "nit",
```

Comment:
Does lerna's `--conventional-commits` understand the what it should do with `nit`?

Reply:
Oop didn't know about that making commitlint less configurable, will be closing PR

Reply:
man I'm so late to this lol -- is the idea that `--conventional-commits` totally bypasses our own config? I'm starting to think we should look into moving away from lerna next half. Either way, would be nice to be able to automatically build the `CHANGELOG` while using our own config

Reply:
Lerna seems abandoned, so moving away from it would be nice.

Looks like from https://yarnpkg.com/features/release-workflow Yarn will get its own release workflow at some point (experimental now). Haven't seen any conventional commit or auto-changelog integrations, though.